nnvm/python/nnvm/frontend/mxnet.py

```diff
@@ -263,6 +263,14 @@ def _expand_dims(inputs, attrs):
     new_attrs['axis'] = _required_attr(attrs, 'axis')
     return _get_nnvm_op(op_name)(*inputs, **new_attrs)
 
+def _lrn(inputs, attrs):
+    op_name, new_attrs = "lrn", {}
+    new_attrs['alpha'] = attrs.get('alpha', 0.0001)
+    new_attrs['beta'] = attrs.get('beta', 0.75)
+    new_attrs['bias'] = attrs.get('knorm', 2)
+    new_attrs['axis'] = attrs.get('axis', 1)
```

Comment:
Could directly assign axis as mxnet never give this attribute.
Also suggest give a hint about why axis is 1.